releasenotes/notes/bipartition_tree-4c1ad080b1fab9e8.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Added a new function :func:`~.bipartition_tree` that takes in spanning tree
+    and a list of populations assigned to each node in the tree and finds all
+    balanced edges, if they exist. A balanced edge is defined as an edge that,
+    when cut, will split the population of the tree into two connected subtrees
+    that have population near the population target within some epsilon. The 
+    function returns a list of all such possible cuts, represented as the set 
+    of nodes in one partition/subtree. For example::
+
+          balanced_node_choices = retworkx.bipartition_tree(
+              tree,
+              pops,
+              float(pop_target),
+              float(epsilon)
+          )
```

Comment:
Can you wrap it in a code block so it looks nice when we render the release notes?

Reply:
Ok, just did this.